src/pages/developing/angular-tutorial/step-5.mdx

```diff
@@ -262,10 +262,10 @@ sure we only deploy immediately after running the build. In the `"scripts"`
 object in `package.json`, add:
 
 ```bash path=package.json
-"deploy": "rm -rf ./dist && yarn build && cf push -f manifest.yml"
+"deploy": "rm -rf ./dist && ng build --prod --aot && cf push -f manifest.yml"
 ```
 
-Next time you want to deploy, you can simply run `yarn deploy`.
+Next time you want to deploy, you can simply run `npm run-script deploy`.
```

Comment:
Same question as above, I believe we use `yarn` everywhere else, so the change is strange. If we want to go with `npm` instead this should be `npm run deploy`

Reply:
In the tutorial, this is the second mention to yarn.

About `npm run deploy`, I didn't try but `npm run-script deploy` works just fine too.

Reply:
the rest of angular tutorial uses ng or npm

Reply:
Ah, my mistake on the `yarn` situation ... This seems fine then